### ISSUE TITLE
node-client not able to create Connection of type "IPCAM"

### ISSUE THREAD

**Issue by siegherr** [OpenVidu/openvidu#599]
**Describe the bug**
In the Session class => `createConnection()` you build the `data`, variable like so:

```javascript
public createConnection(connectionProperties?: ConnectionProperties): Promise<Connection> {
        return new Promise<Connection>((resolve, reject) => {
            // here comes the important part
            const data = JSON.stringify({
                role: (!!connectionProperties && !!connectionProperties.role) ? connectionProperties.role : null,
                data: (!!connectionProperties && !!connectionProperties.data) ? connectionProperties.data : null,
                record: !!connectionProperties ? connectionProperties.record : null,
                kurentoOptions: (!!connectionProperties && !!connectionProperties.kurentoOptions) ? connectionProperties.kurentoOptions : null
            });
```
The problem is the outcoming `data` object does not contain a `type` property.

**Expected behavior**
`createConnection()` should be able to create a Connection of type `IPCAM`.

**Wrong current behavior**
You always end up creating a Connection of type `WEBRTC`

**OpenVidu tutorial where to replicate the error**
This is an EXTREMELY IMPORTANT STEP. If we are able to replicate the error in any of the official [OpenVidu Tutorials](https://github.com/OpenVidu/openvidu-tutorials) or [OpenVidu Demos](https://github.com/OpenVidu), then we will be able to quickly fix it. If you are getting the error in your own application, please try to add the necessary changes to the most similar tutorial so it fails with the same error (try to keep those changes as contained as possible, so that the original tutorial maintains its integrity). Once you have an application to replicate the error, explain in detail the steps to get it like this:
1. Clone repository [...]
2. Compile the application like this [...]
3. Run OpenVidu Server like this [...]
4. Run the application like this [...]
5. Join 1 user... Publish a video stream [...]
6. See error

**OpenVidu deployment info**
How is your OpenVidu Server instance deployed when you get the bug. A couple of possible examples are listed below:
- Docker container as explained in OpenVidu tutorials, run with command `docker run ...` on macOS Catalina 10.15.1
- AWS deployment as explained in OpenVidu Docs (https://docs.openvidu.io/en/stable/deployment/deploying-aws/)

> **IMPORTANT NOTE**: please, if you think the bug might be related to OpenVidu Server side, specify here if you are also getting the error by using OpenVidu Server Demos instance. This instance is publicly available (use it only for this test, because it is not secure!!!): **URL**: `https://demos.openvidu.io:4443`, **SECRET**: `MY_SECRET`

**Client device info (if applicable)**
Describe the client device(s) or platform(s) where you are able to replicate the error. For example:
- Chrome 78.0.3904.97 (Official Build) (64-bit) on Windows 10 (1903).
- Firefox Mobile 68.2.0 running on OnePlus 6 with Android 9.

**Screenshots**

**Additional context**
Add any other context about the problem here. For example, attach any useful logs related to the issue.

**Comment by pabloFuente** [comment]
Hi! Thank you for opening this issue. This has already been addressed in the master branch (commit 0f781f44fda80d22a2263f0620c3ecf80dc17ac2). If you'd like to test the fix, simply compile openvidu-node-client library with the following commands:

```bash
git clone https://github.com/OpenVidu/openvidu.git
cd openvidu/openvidu-node-client
npm install
npm link
```

Then you can install the local openvidu-node-client dependency in your NPM project with `npm link openvidu-node-client`.

**Comment by siegherr** [comment]
Hi, then sorry for opening this issue for no reason! 
Thanks for your suggestion.
